packages/client/src/components/Background/Background.tsx

```diff
@@ -1,16 +1,22 @@
 import { FC, PropsWithChildren } from "react";
 import { Box } from "@mui/system";
 
-import bg from "../../assets/images/game-main-menu-bg.jpg";
+import darkBg from "../../assets/images/dark-theme/game-main-menu-bg.jpg";
+import lightBg from "../../assets/images/light-theme/game-main-menu-bg.jpg";
 
 import global from "../../global/styles/Global.module.scss";
 
-type T_BackgroundProps = PropsWithChildren<{ src?: string; isWhiteBg?: boolean }>;
+type T_BackgroundProps = PropsWithChildren<{ src?: string; isWhiteBg?: boolean; whiteBg?: string }>;
 
-export const Background: FC<T_BackgroundProps> = ({ children, src = bg, isWhiteBg = false }) => {
+export const Background: FC<T_BackgroundProps> = ({ children, src = darkBg, isWhiteBg = false, whiteBg = lightBg }) => {
+  isWhiteBg = localStorage.getItem("theme") === "light";
```

Comment:
Если значение переменной переопределяется здесь, то можно убрать её в пропсах.
```suggestion
  const isWhiteBg = localStorage.getItem("theme") === "light";
```